onnx/backend/test/case/node/averagepool.py

```diff
@@ -199,7 +199,7 @@ def export_averagepool_1d_default() -> None:
         x_shape = np.shape(x)
         kernel_shape = [2]
         strides = [1]
-        out_shape = get_output_shape("VALID", x_shape[2:], kernel_shape, strides)
+        out_shape = get_output_shape("NOTSET", x_shape[2:], kernel_shape, strides)
```

Comment:
Why is this VALID changing to NOTSET? Is this another bug?

Reply:
Good question. It's more like a clarification -- by default pooling operators will use NOTSET instead of VALID and thus the code here looks confusing. However, the result of shape are just accidentally the same here for NOTSET or VALID so no test data needs to be changed. I have mentioned this clarification in the PR description as well.